requirements.txt

```diff
@@ -1,5 +1,5 @@
 cairocffi
-chanjo>=4.7
+chanjo>=4.7.1
```

Comment:
Ahem, 4.7.2 coming up - we never released it.. 😊

Reply:
Right, I assumed that was the latest. That's why i still got the pkg error! 😆